PerformanceTest_Static_PressureDorsal.py

```diff
@@ -217,7 +227,7 @@ def createAvgMat(inputName):
 
         
 
-        outcomes = pd.DataFrame([[subject,config, Movement,
```

Comment:
can we re-add movement to avoid adding for other segments. above you can just add #Movement = 'Sitting' for cycling specific

Reply:
Yes, thank you! I will remove it for the cycling tests and forgot to re-add it. Thanks for the reminder!